lib/read-file.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+
+
+module.exports = function(callback) {
+  fs.readFile(`${__dirname}/../data/one.txt`, function(err, data) {
+    if (err) throw err('did not read one.txt');
+    console.log(data.toString('hex', 0, 8));
+    let result = `${data.toString('hex', 0, 8)} `;
+    return fs.readFile(`${__dirname}/../data/two.txt`, function(err, data) {
+      if (err) throw err('did not read two.txt');
+      console.log(data.toString('hex', 0, 8));
+      result+=`${data.toString('hex', 0, 8)} `;
+      return fs.readFile(`${__dirname}/../data/three.txt`, function(err, data){
+        if (err) throw err('did not read three.txt');
+        console.log(data.toString('hex', 0, 8));
+        result+=`${data.toString('hex', 0, 8)}`;
+        console.log(result);
+        callback(result);
```

Comment:
You are close on the callback pattern, but you are not passing your errors through the callback. Remember the pattern should be callback(err, data). You should also use the callback to pass your errors back instead of throwing them. Think `if (err) callback(err);`